src/components/30-organisms/modal/story.js

```diff
@@ -0,0 +1,87 @@
+import { text, withKnobs } from '@storybook/addon-knobs';
+import { html, render } from 'lit-html';
+import './index';
+
+export default {
+  title: 'Components',
+  decorators: [withKnobs],
+};
+
+export const Modal = () => {
+  setTimeout(() => {
+    document
+      .querySelector('.js-modal-story__button')
+      .addEventListener('click', () => {
+        var modal = document.getElementsByTagName('axa-modal')[0];
+        modal.setAttribute('open', '');
+      });
+  });
```

Comment:
Always search for solutions without "setTimeout". Because this functions depends on how fast the browser renders. And this varies from browser to browser and from device to device. 
And for your specific situation: Cant you set a onClick handler to the button that opens the modal?

Reply:
@danyball I have no idea how to implement this funcationality as you describe it, although I would have liked another approach too. If you could help out, that would be great.